### DISCUSSION TITLE
Where does solidjs stand on the benchmarks shown at https://leptos.dev/?

### DISCUSSION THREAD

**Discussion by hirehamir** [leptos-rs/leptos#2627] [Q&A] [answered]
It'd be kind of nice to see how it holds up against `leptos`.

**Comment by hirehamir** [comment]
https://leptos.dev/

![image](https://github.com/leptos-rs/leptos/assets/43425812/05fd5af1-06a8-46cc-a64e-81dcb8d78e88)

**Comment by gbj** [comment, marked as answer]
Solid's a bit faster -- generally speaking on this benchmark if you compare a JS framework to a WASM one with comparable approaches the JS one will be marginally faster, almost exclusively due to the cost of copying and transcoding UTF-8 strings to UTF-16.

**Reply by hirehamir** [reply]
That makes sense. Thanks for the response.
